tsconfig.json

```diff
@@ -3,9 +3,11 @@
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
     "target": "esnext" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
-    "module": "Node16" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
-    //"erasableSyntaxOnly": true, - we need TS 5.8 first
+    "module": "nodenext" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
+    "erasableSyntaxOnly": true,
     "allowImportingTsExtensions": true,
+    "skipLibCheck": true,
```

Comment:
skipLibCheck to avoid typechecking external `.d.ts` files - that have errors in them - mostly due to gatsby/react/JSX upstream issues :(